### DISCUSSION TITLE
Recommended Model selection for M1+ Mac

### DISCUSSION THREAD

**Discussion by jfbloom22** [chidiwilliams/buzz#957] [General]
I am having decent results with the Whisper (Tiny) model.  The Whisper (small) model does a better job but it is too slow.   What are most people using and finding a good mix of speed and quality on an M1+ Mac?

**Comment by raivisdejus** [comment]
Next version of the Buzz will have support for the Whisper Turbo models. This support is also available in the latest development build https://github.com/chidiwilliams/buzz/actions/workflows/ci.yml?query=branch%3Amain You need to download `Buzz-macOS-ARM64`

If this is fast enough for you it is up to you to decide. 

Also try `Whisper.cpp` engine, it works faster than the regular `Whisper` and may let you run larger models sufficiently fast.
